### PR TITLE
feature/AT-UN-17 css js img - static files added to public folder

### PR DESCRIPTION
**Pull request by Ignacio-jipl97**
feature/AT-UN-17 css js img - static files added to public folder

### CODE REVIEW

File: src/public/js/at-university-util.js [hunk 1]
```diff
@@ -0,0 +1 @@
+/* Archivo en blanco, se valido con el arquitecto */
```
Comment (at-internship) [inline src/public/js/at-university-util.js]: This comment is not neccessary

File: src/public/css/main.css [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+    overflow: hidden;
+}
+
+.embed-container iframe {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+}
```
Comment (at-internship) [inline src/public/css/main.css]: Remove until this line

File: src/public/css/main.css [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+.map-container iframe {
+    left: 0;
+    top: 0;
+    height: 80%;
+    width: 80%;
+    position: relative;
+    max-height: 450px;
+}
+
+.embed-container {
```
Comment (at-internship) [inline src/public/css/main.css]: Remove until this style

File: src/public/css/main.css [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+    width: auto;
+}
+
+#invoicesList.scroll-wrapper-div {
+    overflow-y: scroll;
+    max-height: 200px;
+    width: auto;
+}
+
+.sensitive {
```
Comment (at-internship) [inline src/public/css/main.css]: Remove until this style

File: src/public/css/main.css [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+    background-color: #007bff;
+}
+
+#dashboard.scroll-wrapper-div {
+    overflow-y: scroll;
+    max-height: 600px;
+    width: auto;
+}
+
+#notes.scroll-wrapper-div {
```
Comment (at-internship) [inline src/public/css/main.css]: Remove until this style

File: src/public/css/main.css [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+    overflow-y: scroll;
+    max-height: 200px;
+    width: auto;
+}
+
+#invoicesList.scroll-wrapper-div {
```
Comment (at-internship) [inline src/public/css/main.css]: Remove until this style

File: src/public/css/main.css [hunk 7]
```diff
@@ -0,0 +1,252 @@
...
+a.link-user {
+    color: #373a3c;
+}
+
+a.link-user:hover {
+    color: #373a3c;
+    text-decoration: none;
+}
+
+.amount {
```
Comment (at-internship) [inline src/public/css/main.css]: Remove until this style

File: src/public/css/main.css [hunk 8]
```diff
@@ -0,0 +1,252 @@
...
+}
+
+
+/* reports */
+
+.reports_img {
+    width: 80%;
+}
+
+h2.title-reports {
```
Comment (at-internship) [inline src/public/css/main.css]: Remove until this style

File: src/public/css/main.css [hunk 9]
```diff
@@ -0,0 +1,252 @@
...
+
+.layout--radius {
+    -webkit-border-radius: 50%;
+    border-radius: 50%;
+}
+
+
+/* reports */
+
+.reports_img {
```
Comment (at-internship) [inline src/public/css/main.css]: Remove until this style

File: src/public/css/main.css [hunk 10]
```diff
@@ -0,0 +1,252 @@
...
+    text-align: center;
+}
+
+.layout--radius {
+    -webkit-border-radius: 50%;
+    border-radius: 50%;
+}
+
+
+/* reports */
```
Comment (at-internship) [inline src/public/css/main.css]: Remove until this style

File: src/public/css/main.css [hunk 11]
```diff
@@ -0,0 +1,252 @@
...
+}
+
+.calendar_img {
+    width: 80%;
+}
+
+
+/* HOME */
+
+#home.scroll-wrapper-div {
```
Comment (at-internship) [inline src/public/css/main.css]: Remove until this style

File: src/public/css/main.css [hunk 12]
```diff
@@ -0,0 +1,252 @@
...
+    width: 100%;
+    height: 100%;
+}
+
+.calendar_img {
+    width: 80%;
+}
+
+
+/* HOME */
```
Comment (at-internship) [inline src/public/css/main.css]: Remove until this style

File: src/public/css/main.css [hunk 13]
```diff
@@ -0,0 +1,252 @@
...
+.banners {
+    width: 85%;
+}
+
+.carousel-inner img {
+    width: 100%;
+    height: 100%;
+}
+
+.calendar_img {
```
Comment (at-internship) [inline src/public/css/main.css]: Remove until this style

File: src/public/css/main.css [hunk 14]
```diff
@@ -0,0 +1,252 @@
...
+
+.logo {
+    width: 30%;
+}
+
+.welcome {
+    width: 70%;
+}
+
+.banners {
```
Comment (at-internship) [inline src/public/css/main.css]: Remove until this style

File: src/public/css/main.css [hunk 15]
```diff
@@ -0,0 +1,252 @@
...
+    html {
+        font-size: 14px;
+    }
+}
+
+.logo {
+    width: 30%;
+}
+
+.welcome {
```
Comment (at-internship) [inline src/public/css/main.css]: Remove until this style

**Review by at-internship — CHANGES_REQUESTED** [review]

**Review by at-internship — APPROVED** [review]
Approved

**Review by pinto1997 — APPROVED** [review]

**Pull request merged**